### ISSUE TITLE
User Story: Address lookup and verification

### ISSUE THREAD

**Issue by mthong** [codeforsanjose/upfront-info#6]
As a prospective business owner, I want to enter in the address of my business so that the Upfront Info tool can tell me what permits I need for that location. 

- At minimum, tool should verify that the address is actually in San Jose. If it's not a San Jose address, it should let the user know, and ask for another address.
- At minimum, the user should see the location on a map.  
- Ideally, tool should be able to correct a malformed address or suggest alternate address if some elements are wrong
- Ideally, tool will suggest possible addresses, rather than requiring entire address (city, state, zipcode)

**Comment by CelesteComet** [comment]
Thanks, working on it right now!

**Comment by bvi1994** [comment]
What I could do is set up a dummy page that can do the above. Would that be good @CelesteComet ?

**Comment by CelesteComet** [comment]
yeah that would be awesome, lets use mapbox, they can work with shape files.

**Comment by bvi1994** [comment]
Alright. I'm assuming we are using this: 

https://www.npmjs.com/package/mapbox

https://www.mapbox.com/api-documentation/?language=JavaScript#retrieve-a-static-map-from-a-style
